### PR TITLE
Create missing yamls; delete yaml of inactive target

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/SCAU27181_312779.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - click_on:
+        - value: Submit
+          selector: "#submitButton"
+    - wait:
+        - value: 1
+    - find:
+        - selector: "tr:nth-child(3)"
+  success:
+    body:
+      excludes: "Select an Issue"
```
Comment (tirish) [inline members/SCAU27181_312779.yaml:54]: how confident are you in this exclusion? I think at the very least we should have an `includes` to ensure we aren't on some sort of error page

**Review by vramoscoreas — APPROVED** [review]

File: members/SCAU27181_312779.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/SCAU27181_312779.yaml:54]: I'm pretty confident. FWIW, this the same "template" we've used for all CA yamls. I haven't seen any indication that we might be getting false positives. 

Of course, it's a different story if say the site is down after clicking `submit`, which would gives us a false positive. 

I'm ok with adding a `includes` with this target's thank you msg, tho, if you think that's safer.

Comment (tirish) [reply members/SCAU27181_312779.yaml:54]: generally we should always have an `includes`, even when there is an `excludes` to ensure unexpected failures are guarded against

Comment (OdeclasV) [reply members/SCAU27181_312779.yaml:54]: Cool, added it

**Pull request merged**